pep-9999.rst

```diff
@@ -0,0 +1,293 @@
+and this one is to move the incremental feature delivery period into the
+pre-release phase of development (before strict API and ABI stability
+requirements apply), rather than having it apply to the initial X.Y.Z minor
+releases within a series.
+
+
+Example Future Release Schedules
+================================
+
+Under this proposal, Python 3.9.0b1 would be released in December 2019, two
+months after the Python 3.8.0 baseline feature release in October 2019.
+
+The 3.9.0b2 release would then follow 2 months later in February 2020,
+continuing through to 3.9.0b9 in April 2021.
+
+3.9.0rc1 would be published in June 2021, 3.9.0rc2 in July 2021, and then
+then the full release published as 3.9.0 in August 2021.
+
+The cycle would then begin over again in October 2021, with the publication
+of 3.10.0b1.
+
+Assuming maintenance releases of 3.9.x were also to occur every other month
+(but offset from the 3.10.0 beta releases), then the overall release timeline
+would look like:
+
+* 2019-12: 3.9.0b1
+* ... beta releases every other month
+* 2021-04: 3.9.0b9
+* 2021-06: 3.9.0rc1 (feature freeze, ABI freeze, pyc format freeze)
+* 2021-07: 3.9.0rc2
+* 2021-08: 3.9.0
+* 2021-09: 3.9.1, 3.8.x (final 3.8.x binary maintenance release)
+* 2021-10: 3.10.0b1
+* 2021-11: 3.9.2
+* 2021-12: 3.10.0b2
+* ... beta and maintenance releases every other month
+* 2022-04: 3.10.0b10
+* 2022-05: 3.9.11
+* 2022-06: 3.10.0rc1 (feature freeze, ABI freeze, pyc format freeze)
+* 2022-07: 3.10.0rc2, 3.9.12
+* 2022-08: 3.10.0
+* 2022-09: 3.10.1, 3.9.13 (final 3.9.x binary maintenance release)
+
+
+(The exact schedule of maintenance releases would be up to the release team -
+alternating months with the rolling beta releases is just one possibility)
+
+In this model, there are always two or three active maintenance branches, as in
+the status quo.
+
+
+Proposal
+========
+
+TODO (more details than the abstract, less detail than the design discussion)
+
+Some specific points to cover (either here or in the design discussion):
+
+- automatically appending "-unstable" to SOABI prior to rc1 to help make it
+  clear that only extension modules (and wheel archives) targeting the stable
+  C ABI can be reliably used across the different beta releases
+- potentially consider doing the same for pyc cache files
+- address the pickle forward and backward compatibility question
+- consider changing documentation versionadded/changed notes to include the
+  beta release version (for the benefit of beta release users)
+
+
+Caveats and Limitations
+=======================
+
+Actual release dates may be adjusted up to a month earlier or later at
+the discretion of the release manager, based on release team availability, and
+the timing of other events (e.g. PyCon US, or the annual core development
+sprints). However, part of the goal of this proposal is to provide a consistent
+cadence for both contributors and end users, so adjustments ideally would be
+rare.
```

Comment:
I think this last part could be split into its own sentence.

```
... contributors and end users. So ideally, adjustments would be rare. 
```

Reply:
The whole sentence can probably be reworded to avoid making people work so hard to parse it:

> However, as a goal of the proposal is to provide a consistent release cadence, adjustments should be rare.